src/Tasks/SynTree/Config.hs

```diff
@@ -113,7 +106,4 @@ data SynTreeInst =
     { tree :: SynTree BinOp Char
```

Comment:
I guess the `SynTreeInst` type itself is now superfluous and can be removed completely?